### ISSUE TITLE
Who did it? Windows 10, version 2004 and ReflectDrivers

### ISSUE THREAD

**Issue by Diapolo** [veracrypt/VeraCrypt#633]
I'm not sure if I want to try a Win10 upgrade to 2004 on my productive system yet. So were you successful by using the ReflectDrivers-mechanism to upgrade on a system encrypted system?

**Comment by HCl-not-HCi** [comment]
Worked for me after I figured out how to do this with ReflectDrivers. Updated from 1909

**Comment by wurstpistole** [comment]
I just upgraded from 1909 to 2004 with a mediacreationtool 2004 iso and the reflectdrivers option as instructed and it worked fine. It rebooted five to six times and in between (during the reboot that happens at around 30% of installation) started a file system repair of my (encrypted) system drive, which got me worried for a minute but then continued just fine.

It was the first reflectdrivers update I tried, used to decrypt, upgrade and encrypt for the last years because I had one bad experience a few years ago where I (stubbornly) tried to upgrade without decrypting and lost access to my system. But iirc it was back in Truecrypt times.

Long story short, upgrading worked fine.

**Comment by ColinFinck** [comment]
I have serious problems after upgrading my VeraCrypt-encrypted system to Windows 10 Version 2004: https://github.com/veracrypt/VeraCrypt/issues/640
